creator-node/src/routes/tracks.js

```diff
@@ -210,9 +210,14 @@ module.exports = function (app) {
       } = req.body
 
       // Input validation
-      if (!blockchainTrackId || !blockNumber || !metadataFileUUID) {
+      if (
+        !blockchainTrackId ||
+        !blockNumber ||
+        !metadataFileUUID ||
+        !transcodedTrackUUID
```

Comment:
transcodedTrackUUID doesn't seem to be passed into updateTrack [in libs](https://github.com/AudiusProject/audius-protocol/blob/b0ab3fe3be7192a9e25247ebc32c295d35f2d720/libs/src/api/track.js#L531), this will immediately reject that during the associate step going forward. we should fix that or make it not required here

Reply:
yeah sorry just running maddog tests right now, would have caught this soon
will check

Reply:
made this change, testing now

Reply:
fixed and tested